test/test-span-data.js

```diff
@@ -58,6 +59,23 @@ describe('SpanData', function() {
     });
   });
 
+  it('adds context to headers', function() {
```

Comment:
This spec doesn't read too well. If I am not mistaken this test checks that 'addContextToHeaders adds context to headers'. Current layout of the code makes it read as 'SpanData adds context to headers'.

Can you wrap this section with a `describe('addContextToHeaders', ...)`.